user-interface/src/data-verification/TransferOrderAccordion.tsx

```diff
@@ -145,20 +140,17 @@ export function TransferOrderAccordion(props: TransferOrderAccordionProps) {
       setSuggestedCases(suggestedCases);
     }
     setLoadingSuggestions(false);
+    const button = event.target as HTMLButtonElement;
+    button.classList.add('active');
```

Comment:
I see here that we are adding this class, but I don't think we ever did anything with it.  And I don't see anywhere that we are removing this class.  Maybe we just take these 2 lines out.